packages/babel-plugin-transform-remove-console/__tests__/remove-console-test.js

```diff
@@ -144,3 +145,65 @@ describe("remove-console-plugin", () => {
   `
   );
 });
+
+describe("remove-console-plugin with excludes argument", () => {
```

Comment:
Ah, I was following the model from `babel-plugin-minify-type-constructors` which did not when there were options, but digging into the function prototype looks like I can pass options into `thePlugin` as well, will update.